installation/resources/components.yaml

```diff
@@ -10,7 +10,6 @@ components:
   - name: "istio-resources"
   - name: "logging"
   - name: "telemetry"
-  - name: "tracing"
```

Comment:
for consistency, can you also update the document and double-check if anything left in reconciler needs to be cleanup, e.g: howto readme still mentioned tracing the https://github.com/kyma-incubator/reconciler/blob/0da5d348fab624057b81dca66a076a8791411b54/docs/howto.md?plain=1#L19

Reply:
thanks will remove that as well